pkg/distributor/distributor.go

```diff
@@ -185,9 +185,10 @@ type Config struct {
 	RetryConfig     RetryConfig     `yaml:"retry_after_header"`
 	HATrackerConfig HATrackerConfig `yaml:"ha_tracker"`
 
-	MaxRecvMsgSize           int           `yaml:"max_recv_msg_size" category:"advanced"`
-	MaxRequestPoolBufferSize int           `yaml:"max_request_pool_buffer_size" category:"experimental"`
-	RemoteTimeout            time.Duration `yaml:"remote_timeout" category:"advanced"`
+	MaxRecvMsgSize                 int           `yaml:"max_recv_msg_size" category:"advanced"`
+	MaxOtelDecompressedRecvMsgSize int           `yaml:"max_otel_decompressed_recv_msg_size" category:"experimental"`
```

Comment:
I would suggest to use the correct naming here:

```suggestion
	MaxOtlpRequestSize int           `yaml:"max_otlp_request_size" category:"experimental"`
```

Since Mimir accepts "OTLP/HTTP requests".

The fact that it can be compressed is irrelevant, and it doesn't make much sense to apply limit to compressed message only, so I dropped "Decompressed" part of the name (everywhere, not just here).

Reply:
I agree it should be OTLP, rather than OTel. Style nit, can we write `OTLP` here to be consistent with Go's consistently capitalized initialisms idiom?

Reply:
rename done in 1ba1bcb4854d016dfada97077fc1696ea23c84e3